doc/application/application.rst

```diff
@@ -82,9 +82,23 @@ Source Tree Structure
 Understanding the Zephyr source tree can be helpful in locating the code
 associated with a particular Zephyr feature.
 
-The Zephyr source tree provides the following top-level directories,
-each of which may have one or more additional levels of subdirectories
-which are not described here.
+At the top of the tree there are several files that are of importance:
+
+:file:`CMakeLists.txt`
+    The top-level file for the CMake build system, containing a lot of the
+    logic required to build Zephyr.
+
+:file:`Kconfig`
+    The top-level Kconfig file, which refers to the file :file:`Kconfig.zephyr`
+    also found at the top-level directory.
+
+:file:`west.yml`
+    The :ref:`west` manifest, listing the external repositories required to
```

Comment:
Just minor detail.
I assume one point of splitting up the repos, besides licenses, is to be able to `not` include the repos you don't need for a particular project / system.
So in that case, the text: `required to build zephyr`
might be a little bit misleading.

Reply:
Well, required to build upstream zephyr. I think I will leave it like this for the time being, this text will go through many iterations in the near future for sure :)